src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -151,33 +155,50 @@ def get_deployment_config(app):
     current_prod_env = whodaman()  # current_prod_end = <current-data-env>
     my_env = get_my_env(app)
     deploy_cfg['ENV_NAME'] = my_env
-    if current_prod_env == my_env:
-        log.info('This looks like our production environment -- do not wipe ES')
-        deploy_cfg['WIPE_ES'] = False
-    elif my_env == guess_mirror_env(current_prod_env):
-        log.info('This looks like our staging environment -- do not wipe ES')
-        deploy_cfg['WIPE_ES'] = False  # do not wipe ES
+    deploy_cfg['SKIP'] = False  # set to True to skip the create_mapping step
+    deploy_cfg['STRICT'] = False
+    deploy_cfg['WIPE_ES'] = False
+    if my_env == guess_mirror_env(current_prod_env):
+        log.info('This looks like our staging environment -- wipe ES')
+        deploy_cfg['WIPE_ES'] = True
+        deploy_cfg['STRICT'] = True
     elif is_stg_or_prd_env(my_env):
         log.info('This looks like an uncorrelated production environment. Something is definitely wrong.')
-        exit(0)
+        raise RuntimeError('Tried to run CMOD on production - error\'ing deployment')  # note that this will cause any deployments to production to fail!
```

Comment:
We should probably make a proper error class to use so that the comment won't be needed.
```
class DeploymentFailure(RuntimeError):
    pass
```